examples/platform_services/lib/main.dart

```diff
@@ -13,41 +13,48 @@ class PlatformServices extends StatefulWidget {
 }
 
 class _PlatformServicesState extends State<PlatformServices> {
-  double _latitude;
-  double _longitude;
+  Future<dynamic> _locationRequest;
 
   @override
   Widget build(BuildContext context) {
-    return new Material(
-      child: new Center(
-        child: new Column(
-          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-          children: <Widget>[
-            new Text('Hello from Flutter!'),
-            new RaisedButton(
-              child: new Text('Get Location'),
-              onPressed: _getLocation
-            ),
-            new Text('Latitude: $_latitude, Longitude: $_longitude'),
-          ]
-        )
-      )
-    );
+    return new Material(child: new Center(child: new Column(
```

Comment:
If you add trailing commas everywhere there are permitted and run dartfmt, you'll get code that matches https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo in this example.